OpenAI-DotNet/FineTuning/FineTuneJob.cs

```diff
@@ -8,7 +8,20 @@ namespace OpenAI.FineTuning
     public sealed class FineTuneJob
     {
         [JsonConstructor]
-        public FineTuneJob(string id, string @object, string model, int createdAtUnixTime, IReadOnlyList<Event> events, string fineTunedModel, HyperParams hyperParams, string organizationId, IReadOnlyList<FileData> resultFiles, string status, IReadOnlyList<FileData> validationFiles, IReadOnlyList<FileData> trainingFiles, int updatedAtUnixTime)
+        public FineTuneJob(
```

Comment:
no similar formatting in unity PR?

Reply:
We might as well add the parameter attributes too while we're at it.

Reply:
I copied the formatting from unity branch, its already good.